ui/DesignSystem/Primitive/Text.svelte

```diff
@@ -22,6 +22,9 @@
     font-size: 14px;
     line-height: 20px;
   }
+  .medium {
+    font-family: var(--typeface-medium);
```

Comment:
We should probably also add an explicit `font-size` and `line-height` if this is a new text style.